editor/Dockerfile

```diff
@@ -12,14 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# To build the editor using the current state of ord-schema:
+# $ cd path/to/ord-schema
+# $ docker build --file=editor/Dockerfile -t openreactiondatabase/ord-editor .
+#
+# To test the editor:
+# $ docker run --rm -d -p 5000:5000 --name editor openreactiondatabase/ord-editor
+# $ node editor/js/test.js
+# $ docker stop editor
+#
+# To push the built image to Docker Hub:
+# docker push openreactiondatabase/ord-editor
+
 FROM continuumio/miniconda3
-WORKDIR /usr/src/app
 
 RUN apt-get update && apt-get install --yes build-essential procps unzip
 RUN conda install -c rdkit rdkit
 RUN conda install flask nodejs
 
 # Fetch/build editor dependencies.
+# NOTE(kearnes): Do this before COPYing the local state so it can be cached.
+WORKDIR /usr/src/app/ord-schema/editor
 RUN git clone https://github.com/Open-Reaction-Database/ketcher.git
```

Comment:
Does this mean we are cloning and building ketcher every time the test runs? Or is caching possible?

I'm hoping this test can run quickly.

Reply:
The test currently runs in ~5 min. The only way to cache on GitHub AFAIK would be to create another image that includes all the steps up to getting the local state and having the Dockerfile inherit from that instead of the current miniconda3 image. We could certainly do this.